concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1932,11 +1935,16 @@ jobs:
                   ln -snf $(pwd)/test-config/run /var/vcap/jobs/acceptance-tests/bin/run
                   ln -snf /usr/local/go /var/vcap/packages/golang1.6
                   ln -snf $(pwd)/cf-release/src/github.com/cloudfoundry/cf-acceptance-tests /var/vcap/packages/acceptance-tests/src/github.com/cloudfoundry/
-                  /var/vcap/jobs/acceptance-tests/bin/run
-
+                  if  [ "$ENABLE_CF_ACCEPTANCE_TESTS" != "false" ]; then
+                    /var/vcap/jobs/acceptance-tests/bin/run
+                  else
+                    echo "WARNING: The acceptance tests has been disabled. Set ENABLE_HEALTHCHECK_DB=true when uploading the pipelines to enable them. You can still hijack this container to run them manually"
```

Comment:
I guess "Set ENABLE_HEALTHCHECK_DB=true when uploading the pipelines to enable them." is a copy paste error?